packaging/pre_build_script.sh

```diff
@@ -7,8 +7,28 @@ python3 -m pip install pyyaml
 
 yum install -y ninja-build gettext
 
-wget https://github.com/bazelbuild/bazelisk/releases/download/v1.17.0/bazelisk-linux-amd64 \
-    && mv bazelisk-linux-amd64 /usr/bin/bazel \
+BAZEL_PLATFORM="amd64"
+
+if [[ $(uname -m) == "aarch64" ]]; then
+    BAZEL_PLATFORM=arm64
+    rm -rf /opt/openssl # Not sure whats up with the openssl mismatch
+    # aarch64 does not have envsubst pre-installed in the image, install it here
+    curl -L  https://github.com/a8m/envsubst/releases/download/v1.4.2/envsubst-Linux-arm64 -o envsubst \
+    && mv envsubst /usr/bin/envsubst && chmod +x /usr/bin/envsubst
+    # install cuda 12.8 only for aarch64 for now
+    VER="12-8"
```

Comment:
We had this above as well. Do we need both? maybe we can isolate this in its own script?

Reply:
Also pull cuda from CU_VERSION